### PR TITLE
Math magician calculator

### PR DESCRIPTION
**Pull request by mwafrika**
# All the following features have been implemented successfully:

Delete all the boilerplate from CRA (text, images, styles). ✔️ 
Create a directory called components. ✔️ 
Inside components create a new Calculator.js file. ✔️ 
In Calculator.js, create a React component that matches this design ✔️ 
Don't add any extra design. ✔️ 
Develop just the UI of the calculator; the math logic to make it work will be added in the next project. ✔️ 
Use class-based components. ✔️ 
Once the Calculator is ready, import it and use it in your main component (App.js). ✔️ 
Test it in the browser; it should render without problems. ✔️ 

## [Live Demo](https://mwafrika.github.io/math_magicians/)

calc

Thank you so much for taking the time to review my pull request 💯 🥇 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,50 @@
...
+
 ### Install
 
+`npm install`
+
 ### Usage
 
 ### Run tests
 
 ### Deployment
```
Comment (TedLivist) [inline README.md]: - [ ] In your README, kindly add instructions on how to run the app locally i.e. `npm start`.

- [ ] Remove unnecessary sections from your README that have no descriptive body yet.

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (TedLivist) [inline src/App.js]: - [ ] Your components should be class-based. Kindly change this App component to be a class-based one.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @mwafrika,

Good job so far!
✔️ Good documentation
✔️ App design follows template
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] **You are expected to have deleted the following CRA generated boilerplate files as they won't be used. Kindly delete them.**

  src folder | public folder
  :---:|:---:
  mwafrika-src | mwafrika-public

- [ ] **_Check the comments under the review._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (mwafrika) [reply src/App.js]: Sure let me do it right now. Thanks @TedLivist

**Review by rustam-tolipov — APPROVED** [review]
## Hello @mwafrika 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**